dashboard/test/ui/features/artist.feature

```diff
@@ -20,6 +21,7 @@ Scenario: Winning the first level
   And I wait to see ".congrats"
   And element ".congrats" is visible
   And I press "continue-button"
+  Then I close the dialog
```

Comment:
Got it. Looking in to the step definition, one thing to watch for will be slightly changed wait time behavior between pages. 

The "Then I close the dialog" step uses "When I press ___" under the hood, which calls `wait_with_short_timeout`, which would wait 30 seconds to find the desired element — whereas the "Then I wait to see a dialog" which would previously be the first action after page load waits the default amount (2 minutes).

30 seconds should be enough for a page traversal, & if we see it fail here we can add a step with a longer max wait.